src/BenchmarkDotNet/Running/BenchmarkRunnerClean.cs

```diff
@@ -363,9 +363,16 @@ private static ImmutableArray<ValidationError> Validate(params BenchmarkRunInfo[
                         .Validate(new ValidationParameters(joinedCases, null))
                     );
             }
-
+            var sdkValidator = SdkValidator.Instance;
             foreach (var benchmarkRunInfo in benchmarks)
-                validationErrors.AddRange(benchmarkRunInfo.Config.GetCompositeValidator().Validate(new ValidationParameters(benchmarkRunInfo.BenchmarksCases, benchmarkRunInfo.Config)));
+            {
+                var validators = benchmarkRunInfo.Config.GetValidators().ToList();
+                validators.Add(sdkValidator);
+                var compositeValidator = new CompositeValidator(validators.ToImmutableHashSet());
+
+                var currentValidationErrors = compositeValidator.Validate(new ValidationParameters(benchmarkRunInfo.BenchmarksCases, benchmarkRunInfo.Config));
+                validationErrors.AddRange(currentValidationErrors);
+            }
```

Comment:
This looks like a very strange place to add the validator. I think you should instead add the validation to `CsProjCoreToolchain.Validate`. (And other toolchains that need it. Maybe it would be useful to make them inherit a common base class with that validation.)